Elevel/Application/Features/AuditionCommands/DeleteAuditionCommand.cs

```diff
@@ -0,0 +1,52 @@
+﻿using AutoMapper;
+using Elevel.Application.Interfaces;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.AuditionCommands
+{
+    public class DeleteAuditionCommand
+    {
+        public class Request : IRequest<Response>
+        {
+            public Guid Id { get; set; }
+        }
+
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IApplicationDbContext _context;
+            private readonly IMapper _mapper;
+
+            public Handler(IApplicationDbContext context, IMapper mapper)
+            {
+                _context = context;
+                _mapper = mapper;
+            }
+
+            public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
+            {
+                var audition = await _context.Auditions.FirstOrDefaultAsync(x => x.Id == request.Id, cancellationToken);
+                if (audition is null)
+                {
+                    return null;
```

Comment:
should be consistent, there are many places with throw new NotFoundException

Reply:
Done